spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -878,10 +886,21 @@
       let(:subject_name_subject_type){ :subject_name }
       let(:expected_lcnaf_pid) { ["id": "http://id.loc.gov/authorities/names/n90699999"] }
 
+
       it "returns metadata for term" do
         expect(invenio_rdm_record_converter.send(:subjects_for_scheme, subject_name_terms, subject_name_subject_type)).to eq(expected_lcnaf_pid)
       end
     end
+
+    context "tag scheme" do
```

Comment:
This was just to help myself understand what was happening when the tag field was passed to this function.

Reply:
I see this test was just for that (not for duplication testing). :+1: